src/inputs/ToggleChipGroup.tsx

```diff
@@ -1,13 +1,16 @@
 import { useRef } from "react";
 import { useCheckboxGroup, useCheckboxGroupItem, useFocusRing, VisuallyHidden } from "react-aria";
 import { CheckboxGroupState, useCheckboxGroupState } from "react-stately";
+import { maybeTooltip } from "src/components";
 import { Label } from "src/components/Label";
 import { Css } from "src/Css";
 import { useTestIds } from "src/utils/useTestIds";
 
 type ToggleChipItemProps = {
   label: string;
   value: string;
+  isDisabled?: boolean;
+  disabledReason?: string;
```

Comment:
Suggestion: Instead of two separate props, we typically model this as a union, like:

```
export interface BeamButtonProps {
  /**
   * Whether the interactive element is disabled.
   *
   * If a ReactNode, it's treated as a "disabled reason" that's shown in a tooltip.
   */
  disabled?: boolean | ReactNode;
```

That way the user can't do something like `isDisabled=false disabledReason="some error"` and it's ambiguous that maybe we really be disabled, like does a set `disabledReason` override the `isDisabled` is false?

Reply:
Thats true. Had tunnel vision on the filter so hadn't considered `ToggleChipGroup` ever using anything besides a string